kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -131,7 +131,9 @@ class BinBasedFastPointLocator
                          array_1d<double, TDim + 1 > & N,
                          Element::Pointer& pelement,
                          ResultIteratorType result_begin,
-                         const unsigned int MaxNumberOfResults)
+                         const unsigned int MaxNumberOfResults,
+                         const double Tolerance = 1.0e-3
```

Comment:
When I use IsInside method (perhaps it has nothing to do with what Vicente is doing here), I use the default tolerance suggested by Pooyan (std::numeric_limits<double>::epsilon()) and, in case I do not find where my node falls, I call IsInside again with a higher tolerance of 1.0e-5.